build.gradle

```diff
@@ -121,7 +121,7 @@ ext {
 }
 
 group = "com.amazon.opendistroforelasticsearch"
-version = "${opendistroVersion}.0"
+version = "${opendistroVersion}.1"
```

Comment:
Do we need to increase this in this PR? Can we do it in follow up PR as need to update release notes etc. too

Reply:
actually yes, to distinguish from previous version so that skipExecution can actually works